apache-arrow/README.md

```diff
@@ -1,3 +1,12 @@
+### Why We Mirror Apache Arrow
+
+Groonga depends on Apache Arrow. However, the official Apache Arrow repository has been unstable at times, causing issues for Groonga users. To resolve this situation, we have been providing a mirrored version for Groonga users.
```

Comment:
```suggestion
Groonga depends on Apache Arrow. However, the official Apache Arrow repository has been unstable at times, causing issues for Groonga users.
To resolve this situation, we have been providing a mirrored version for Groonga users.
```

Reply:
fix: [08aecc8](https://github.com/groonga/packages.groonga.org/pull/40/commits/08aecc8f6f8f34e33663114fb072eaeaedba1746) Thanks. I assigned one line to each sentence for consistency.